### ISSUE TITLE
Unit tests succeed but for the wrong reason

### ISSUE THREAD

**Issue by mtmorgan** [Bioconductor/GenomicRanges#8]
The test here https://github.com/Bioconductor/GenomicRanges/blob/master/inst/unitTests/test_GRangesList-class.R#L103 and immediately following fails because it is trying to assign to an object that does not exist outside the assignment

```
> start(GRangesList()) = NULL
Error in start(GRangesList()) = NULL : 
  invalid (NULL) left side of assignment
```

Probably the intention was

```
> grl = GRangesList()
> start(grl) = NULL
Error in `start<-`(`*tmp*`, value = NULL) : 
  replacement 'value' is not an IntegerList with the same elementNROWS as 'x'
```

**Comment by lawremi** [comment]
What's also weird (and irrelevant) is that somehow the `<-` in the code were converted into `=` in your quote of it.

**Comment by hpages** [comment]
Can't believe we had this bug since the very beginning of GenomicRanges. Fixed in beebde3790d3558c596f89bbdf0d816bfc2ca661
